app/assets/stylesheets/articles.scss

```diff
@@ -680,10 +680,9 @@
     &.big-article {
       padding: 0px;
       .picture {
-        padding-top: 42%;
+        padding-top: 14%;
```

Comment:
I feel like we can start with _just_ the article show page and remove this and the HTML adjustment. The index pages are less important for SEO, we _may_ want to do it but for the sake of making this PR simpler we might just want to focus on the show view for now.

Reply:
Okay, sounds reasonable to me! I'll revert these changes. 👍